views/fe-utils/fetchUtil.js

```diff
@@ -88,26 +91,29 @@ const createStudyDetails = async (body) => {
 }
 
 const deleteStudyDetails = async (id) => {
-  const res = await window.fetch(`${routes.basePath}/api/v1/study-details/${id}`, {
-    ...defaultApiOptions,
-    method: 'DELETE',
-  })
+  const res = await window.fetch(
+    `${routes.basePath}/api/v1/study-details/${id}`,
+    {
+      ...defaultApiOptions,
+      method: 'DELETE',
+    }
+  )
 
   if (res.status !== 200) return new Error(res.message)
 
   return res.json()
 }
 
 const createChart = async (formValues) => {
-    const res = await window.fetch(`${routes.basePath}/api/v1/charts`, {
-      ...defaultApiOptions,
-      method: 'POST',
-      body: JSON.stringify(formValues)
-    })
+  const res = await window.fetch(`${routes.basePath}/api/v1/charts`, {
```

Comment:
Here's another case where we can use the route constants.

```suggestion
  const res = await window.fetch(routes.charts, {
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/118/commits/a02b5f9495eed90745bb37c6f90df4df8e963870